src/js/Tabs/__test__/on.test.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import Tabs from '../index';
+import {getByText, fireEvent} from '@testing-library/dom';
+
+// const messages = {
+//   INVALID_ARGUMENT: 'Error: invalid function arguments',
+//   MISSING_TAB_NAME: 'Missing tab name on tab item[{{index}}]',
+//   INVALID_ACTION: 'Behavior invalid',
+//   MISSING_NEW_ITEM_TABNAME: 'Missing tab name.'
+// };
+
+describe('Unit test Tabs disableItem', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('on is called correctly', () => {
+    const items = [{
+      tabName: 'Tab1'
+    }, {
+      tabName: 'Tab2'
+    }];
+    const myTabs = new Tabs({items});
+    const container = myTabs.render();
+    myTabs.on('clickTabItem', (tabIndex) => {
+      expect(tabIndex).toEqual(1);
```

Comment:
Maybe this can not effected.
Need to check  myTabs.on('clickTabItem', (tabIndex) => {}) was fired first.